cliff_detector:
  ros__parameters:
    output_frame_id: camera_depth_frame
    rate: 10
    range_min: 0.5
    range_max: 5.0
    sensor_mount_height: 0.47
    sensor_tilt_angle: 12.0

    publish_depth: true
    cam_model_update: false
    used_depth_height: 400
    block_size: 16
    block_points_thresh: 3
    depth_img_step_row: 8
    depth_img_step_col: 8
    ground_margin: 0.05
    block_row_start: 7
    block_col_start: 0
    block_row_end: 15
    block_col_end: 10000